### ISSUE TITLE
Suggestion: A way of packing .GMA files without Uploading to Workshop

### ISSUE THREAD

**Issue by 0xAA289** [WilliamVenner/gmpublisher#187]
As the title says, there should be an easy and convenient way of Packing a GMA file without uploading to the workshop

**Comment by WilliamVenner** [comment]
There's no need for this, GMA files have no use other than for publishing. They're redundant.

**Comment by 0xAA289** [comment]
Well you have ways of extracting them, 
even a simple way to construct them just so people can finally ditch GMPU for interacting with gma files

**Comment by 0xAA289** [comment]
Not many people do this, but people self host the gma's for manual mounting and it's getting more common on servers

**Comment by WilliamVenner** [comment]
Yeah that's a good point. However, for something like server content I think it'd be more appropriate for people to automate it, and gmpublisher is not really designed for automation, being a GUI app.

**Comment by 0xAA289** [comment]
True that, suggestion still stands though

**Comment by 0xAA289** [comment]
Also considering your implementation of making `.gma` files is blazingly fast compared to `gmad.exe` a CLI util for this would be phenomenal

**Comment by WilliamVenner** [comment]
https://github.com/WilliamVenner/fastgmad
